docs/rpc-compatibility.md

```diff
@@ -0,0 +1,295 @@
+# RPC Updates to the bcoin
+
+## Ports
+Bitcoin Core does not separate wallet and node:
```

Comment:
I would phrase as "Bcoin separates the walled and node RPC ports." from the perspective of describing bcoin.